docs/src/design-system/typography/TextStylesPage.tsx

```diff
@@ -34,37 +34,38 @@ const rows = [
   createData('Caption', 'Caption', 'Light Italic', 'Left', '11px', 'Sentence'),
 ];
 
-const tableCellStyles = (palette: Palette) => {
-  return {
-    width: '7.625rem',
-    height: '2.5rem',
-    borderColor: { dark: palette.grey['600'], light: palette.grey['200'] }[palette.mode],
-  };
+const tableCellStyles = {
+  width: '7.625rem',
+  height: '2.5rem',
 };
 const tableColumns: string[] = [
   'Category',
   'Style Name',
   'Font Styles',
   'Alignment',
   'Size',
-  'Capitalization',
+  'Case',
 ];
 
 const TextStylesPage: React.FC = () => {
   const { palette } = useTheme();
   return (
     <Grid container>
       <Grid item xs={12}>
-        <TopBar title={'STYLE'} />
+        <TopBar title={'Style'} />
       </Grid>
       <TableContainer>
         <Table
           sx={{
+            [`& .${tableCellClasses.root}`]: {
+              p: '0',
+              verticalAlign: 'middle',
+            },
             width: '45.75rem',
-            margin: '2.4375rem auto 22rem',
+            margin: '2.5rem auto 22rem',
+            borderBottom: 1,
+            borderColor: palette.divider,
```

Comment:
I'd expect the `Table` by default with out theme applied would already have a borderBottom and borderColor as specified, no?

Here is the default table with our theme:
![image](https://user-images.githubusercontent.com/10551665/200913049-14d95b33-d99a-4ca3-b7ec-5696f6f16b4a.png)

Reply:
Yes, by default table already have a borderBottom and borderColor, but the last row of borderBottom of the tableBody was 
 also removed by our theme, which was implemented in the `MuiTableBody.ts` I think. So to be consistent with the design, I add a borderBottom specially for the table.
![image](https://user-images.githubusercontent.com/56696843/200916636-390c9dfd-84e6-4390-a0ba-dd2597e8dbc6.png)

Reply:
Okay just looked more closely at the designs - in this case, I think the design is wrong for being inconsistent, but anyways this is fine.